README.md

```diff
@@ -200,9 +200,11 @@ matchingRules:
 - type: PromQL
   promql:
     promql:
-      group(sre:telemetry:managed_labels{_id="",sre="true"})
-      or
-      0 * group(cluster_version{_id=""})
+      topk(1,
```

Comment:
Do I understand it correctly?
`topk(1, ...)` is there because we want to ensure that only ONE sample is returned if there are many.
We did not use it before because many-value is introduced with `by (_id, sre)`.